lang/package.go

```diff
@@ -101,6 +113,15 @@ func PackageWithRepositoryOverrides(repo *Repo) PackageOption {
 	}
 }
 
+// PackageWithIncludeFiles can be used along with the NewPackageFromBuild function to specify
+// a list of files which should only be used for generation.
```

Comment:
We favour active over passive language.

```suggestion
// PackageWithIncludeFiles allows to specify a list of files which should only be used for generation. This function should be used with NewPackageFromBuild.
```